client/dashboard/profile-wizard/steps/theme/preview.js

```diff
@@ -35,6 +40,13 @@ class ThemePreview extends Component {
 		this.state = {
 			device: 'desktop',
 		};
+
+		this.handleDeviceClick = this.handleDeviceClick.bind( this );
+	}
+
+	handleDeviceClick( device ) {
+		recordEvent( 'storeprofiler_store_theme_demo_device', { device } );
```

Comment:
I think it would be valuable to know which theme is viewed via the device as well.

Reply:
Agreed- good catch @justinshreve!  Added in https://github.com/woocommerce/woocommerce-admin/pull/2715/commits/db7d6621e43e5c2e79f81e5f9a6dfef25ec067f6

Thanks for the review 😄